### PR TITLE
remove lambda function as inputs, use lambdaOp instead

### PR DESCRIPTION
**Pull request by vbvg2008**

### CODE REVIEW

File: fastestimator/op/op.py [hunk 1]
```diff
@@ -128,4 +128,4 @@ def __init__(self,
         self.in_list = True
 
     def forward(self, data: List[Any], state: Dict[str, Any]) -> Any:
-        return self.fn(*data)
+        return self.fn(*to_list(data))
```
Comment (TortoiseHam) [inline fastestimator/op/op.py:131]: no need to call to_list when in_list = True

Comment (vbvg2008) [reply fastestimator/op/op.py:131]: When inputs is None, the data becomes None.

Comment (TortoiseHam) [reply fastestimator/op/op.py:131]: that's a problem - should be resolved elsewhere such that in_list = True always results in a list being passed

Comment (vbvg2008) [reply fastestimator/op/op.py]: like this?

File: fastestimator/pipeline.py [hunk 2]
```diff
@@ -21,15 +21,15 @@
...
-from torch.utils.data.dataloader import default_collate
-
 from fastestimator.dataset.batch_dataset import BatchDataset
 from fastestimator.dataset.op_dataset import OpDataset
 from fastestimator.op.numpyop.numpyop import NumpyOp, forward_numpyop
+from fastestimator.op.op import LambdaOp
 from fastestimator.schedule.schedule import Scheduler, get_current_items
 from fastestimator.util.traceability_util import traceable
 from fastestimator.util.util import pad_batch, to_list, to_set
+from torch.utils.data import DataLoader, Dataset, RandomSampler
```
Comment (TortoiseHam) [inline fastestimator/pipeline.py:31]: import sort misconfigured

Comment (vbvg2008) [reply fastestimator/pipeline.py]: done

**Pull request merged**